### PR TITLE
Add openshift e2e testing to pipeline enabled through linchpin libvirt container

### PR DESCRIPTION
**Pull request by samvarankashyap**
$Subject 
Needs few changes (Post review )

### CODE REVIEW

**Comment by scoheb** [comment]
[test]

**Comment by samvarankashyap** [comment]
[test]

File: JenkinsfileStageTrigger [hunk 1]
```diff
@@ -24,6 +24,7 @@ tagMap['rpmbuild'] = STABLE_LABEL
 tagMap['rsync'] = STABLE_LABEL
 tagMap['ostree-compose'] = STABLE_LABEL
 tagMap['package-test'] = STABLE_LABEL
+tagMap['linchpin-libvirt'] = "latest"
```
Comment (scoheb) [inline JenkinsfileStageTrigger]: should be STABLE_LABEL

File: config/s2i/create-containers.sh [hunk 2]
```diff
@@ -3,10 +3,11 @@ set -x
 ##
 oc project continuous-infra
 oc create -f rpmbuild/rpmbuild-buildconfig-template.yaml
-oc create -f rpmbuild/rsync-buildconfig-template.yaml
+oc create -f rsync/rsync-buildconfig-template.yaml
```
Comment (scoheb) [inline config/s2i/create-containers.sh]: thanks!

**Comment by scoheb** [comment]
you also need to add an entry to JenkinsfileStageMerge for your image:

imageList = ["jenkins-continuous-infra-slave", "rpmbuild", "ostree-compose", "rsync"]

File: config/s2i/linchpin_libvirt/linchpin_libvirt-buildconfig-template.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          - name: CONTAINER_NAME
+            value: linchpin-libvirt
+      type: Docker
+    triggers:
+    - type: ConfigChange
+parameters:
+- description: Git repository with Dockerfile and slave entrypoint.
+  displayName: Repository URL
+  name: REPO_URL
+  value: https://github.com/samvarankashyap/ci-pipeline.git
```
Comment (scoheb) [inline config/s2i/linchpin_libvirt/linchpin_libvirt-buildconfig-template.yml]: should be main repo

**Comment by samvarankashyap** [comment]
[test]

**Comment by samvarankashyap** [comment]
[test]

**Comment by samvarankashyap** [comment]
[test]

**Comment by samvarankashyap** [comment]
[test]

**Pull request closed without merging**